content/en/project/getInvolved/_index.md

```diff
@@ -28,40 +28,44 @@ Bug reports, feature requests, documentation problems, and questions are posted
 
 * [Medley](https://github.com/Interlisp/medley): Lisp code, build scripts and GitHub automation
 * [maiko](https://github.com/Interlisp/maiko): The C-coded virtual machine
-* [this web site](https://github.com/Interlisp/Interlisp.github.io#README)
+* [the Interlisp.github.io repository](https://github.com/Interlisp/Interlisp.github.io#README) of the interlisp.org web site
 
 ## 3. Join in the discussions
 
 ### Discussion board
 
 The [discussion board on GitHub](https://github.com/orgs/Interlisp/discussions) is for general discussions and questions on the Medley environment, the Interlisp language, and Lisp development on the Medley system.
 
-### Socials
+### Social media accounts
 
-You can follow us on the Fediverse and LinkedIn.
+You can follow our official social media accounts which we monitor.
```

Comment:
I'd drop 'which we monitor'.  I think it's clear, especially since below we mention we don't monitor the platform formerly known as Twitter.